functions.py

```diff
@@ -2,159 +2,159 @@
-        ext = raw_input('Save as: ')
-        im = Image.open(directory)
-        if not os.path.exists('formated_images'):
-            os.mkdir('formated_images')
-        try:
-            print "Name: " + os.path.splitext(ext)[0]
-            print "Format: " + os.path.splitext(ext)[1]
-            im.save('formated_images/'+ext)
-            raw_input('Saved in formated_images.')
-        except:
-            raw_input('Error..')
+            if not os.path.exists('formated_images'):
+                os.mkdir('formated_images')
+            try:
+                print "Name: " + os.path.splitext(ext)[0]
+                print "Format: " + os.path.splitext(ext)[1]
+                im.save('formated_images/'+ext)
+                raw_input('Saved in formated_images.')
+            except:
+                raw_input('Error..')
 
-    elif choice == "2":
-        f_directory = raw_input('Directory to folder: ')
-        ext = raw_input('Format: ')
-        try:
-            fieldlist = os.listdir(f_directory)
-        except OSError:
-            raw_input('Folder does not exist.')
+        elif choice == "2":
+            f_directory = raw_input('Directory to folder: ')
+            ext = raw_input('Format: ')
+            try:
+                fieldlist = os.listdir(f_directory)
+            except OSError:
+                raw_input('Folder does not exist.')
+                sys.exit()
+            if not os.path.exists('formated_images'):
+                os.mkdir('formated_images')
+            print "Format: " + ext
+            fieldlist_range = len(fieldlist)
+            for item in fieldlist:
+                directory = f_directory+ "\\" + item
+                im = Image.open(directory)
+                try:
+                    im.save('formated_images/'+os.path.splitext(item)[0]+'.'+ext)
+                    print "- " + os.path.splitext(item)[0] + " : " + "DONE"
+                    fieldlist_range-=1
+                    if fieldlist_range == 0:
+                        raw_input('Saved in formated_images.')
+                except:
+                    raw_input('Error..')
+        elif choice == "3":
+            os.system('cls')
             sys.exit()
-        if not os.path.exists('formated_images'):
-            os.mkdir('formated_images')
-        print "Format: " + ext
-        fieldlist_range = len(fieldlist)
-        for item in fieldlist:
-            directory = f_directory+ "\\" + item
+        else:
+            raw_input('Error')
+            sys.exit()
+            os.system('cls')
+
+    def make_thumbnail(self):
+        print '''
+        =====================
+        Make thumbnail
+        =====================
+        1. File
+        2. Folder
+        3. Close
+        =====================
+        '''
+        choice = raw_input(">> ")
+        if choice == "1":
+            directory = raw_input('Directory to file: ')
+            name = raw_input('Save as: ')
```

Comment:
Input handling should be extracted to `Main` class. Here we should just do operations on the images not read the input.